code/modules/jobs/job_types/roguetown/courtier/councillor.dm

```diff
@@ -51,6 +51,23 @@
 /datum/outfit/job/roguetown/councillor
 	job_bitflag = BITFLAG_ROYALTY
 
+/datum/job/roguetown/lord/after_spawn(mob/living/H, mob/M, latejoin = TRUE)
```

Comment:
This is the Councillor file. This should be roguetown/councillor if you want them to have Lord/Lady.